tgui/packages/tgui/interfaces/common/RecipeLookup.js

```diff
@@ -0,0 +1,210 @@
+import { Box, LabeledList, Chart, Flex, Button, Icon } from '../../components';
+import { useBackend } from '../../backend';
+
+export const RecipeLookup = (props, context) => {
+  const { recipe } = props;
+  const { act } = useBackend(context);
+  return (
+        
+    recipe && (
+      <LabeledList labelBold>
+        <LabeledList.Item bold label="Recipe">
+          <Icon name="circle" mr={1} color={recipe.reagentCol} />
+          {recipe.name}
+          <Button
+            key={"reduce_index"}
+            icon="arrow-left"
+            ml={3}
+            disabled={recipe.subReactIndex
+                            === 1 ? true : false}
+            content={null}
+            onClick={() => act('reduce_index', {
+              id: recipe.name,
+            })} />
+          <Button
+            key={"increment_index"}
+            icon="arrow-right"
+            disabled={recipe.subReactIndex
+                            === recipe.subReactLen?true:false}
```

Comment:
Are you serious?

`? true : false` moment.

```suggestion
            disabled={recipe.subReactIndex === recipe.subReactLen}
```